README.md

```diff
@@ -34,15 +34,18 @@ regarding Chainlink social accounts, news, and networking.
 
 1. [Install Go 1.22](https://golang.org/doc/install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
    - Example Path for macOS `export PATH=$GOPATH/bin:$PATH` & `export GOPATH=/Users/$USER/go`
+   - Note that using Go 1.23, the current install at the above path, will result in 2 errors the first of which is `undefined: runtime_cyclesPerSecond`
 2. Install [NodeJS v20](https://nodejs.org/en/download/package-manager/) & [pnpm v9 via npm](https://pnpm.io/installation#using-npm).
    - It might be easier long term to use [nvm](https://nodejs.org/en/download/package-manager/#nvm) to switch between node versions for different projects. For example, assuming $NODE_VERSION was set to a valid version of NodeJS, you could run: `nvm install $NODE_VERSION && nvm use $NODE_VERSION`
 3. Install [Postgres (>= 12.x)](https://wiki.postgresql.org/wiki/Detailed_installation_guides). It is recommended to run the latest major version of postgres.
    - Note if you are running the official Chainlink docker image, the highest supported Postgres version is 16.x due to the bundled client.
    - You should [configure Postgres](https://www.postgresql.org/docs/current/ssl-tcp.html) to use SSL connection (or for testing you can set `?sslmode=disable` in your Postgres query string).
 4. Ensure you have Python 3 installed (this is required by [solc-select](https://github.com/crytic/solc-select) which is needed to compile solidity contracts)
-5. Download Chainlink: `git clone https://github.com/smartcontractkit/chainlink && cd chainlink`
-6. Build and install Chainlink: `make install`
-7. Run the node: `chainlink help`
+5. Ensure you have jq installed (otherwise `tools/bin/ldflags: line 6: jq: command not found` will be reported in `make install`)
+6. Download Chainlink: `git clone https://github.com/smartcontractkit/chainlink && cd chainlink`
+7. Install Chainlink: `make install`
+8. Build Chainlink: `make chainlink`
```

Comment:
IFIRC you don't need to do both 7 and 8. One will install so that you can call the binary like `chainlink`, and the other makes it so you can call the binary like `./chainlink`

Reply:
Where does the binary from `make install` live and what should PATH contain?  I only saw a chainlink file in /chainlink dir if I ran make chainlink, and `chainlink` returns `zsh: command not found: chainlink`

Reply:
@patrickhuie19 ping

Reply:
@patrickhuie19 ping

Reply:
https://pkg.go.dev/cmd/go

> Compile and install packages and dependencies [¶](https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies)
> 
> Usage:
> 
> go install [build flags] [packages]
> 
> Install compiles and installs the packages named by the import paths.
> 
> Executables are installed in the directory named by the GOBIN environment variable, which defaults to $GOPATH/bin or $HOME/go/bin if the GOPATH environment variable is not set. Executables in $GOROOT are installed in $GOROOT/bin or $GOTOOLDIR instead of $GOBIN.